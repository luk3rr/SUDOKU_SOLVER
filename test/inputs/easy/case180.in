520700000 000180000 010030270 000000009 860000000 009060802 000004001 600050004 002010068 
