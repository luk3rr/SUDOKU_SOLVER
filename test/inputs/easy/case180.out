Solving the following grid:
5 2 0 | 7 0 0 | 0 0 0 
0 0 0 | 1 8 0 | 0 0 0 
0 1 0 | 0 3 0 | 2 7 0 
------+-------+------
0 0 0 | 0 0 0 | 0 0 9 
8 6 0 | 0 0 0 | 0 0 0 
0 0 9 | 0 6 0 | 8 0 2 
------+-------+------
0 0 0 | 0 0 4 | 0 0 1 
6 0 0 | 0 5 0 | 0 0 4 
0 0 2 | 0 1 0 | 0 6 8 


Solution found :')

5 2 4 | 7 9 6 | 1 8 3 
7 3 6 | 1 8 2 | 4 9 5 
9 1 8 | 4 3 5 | 2 7 6 
------+-------+------
2 4 5 | 8 7 1 | 6 3 9 
8 6 3 | 2 4 9 | 5 1 7 
1 7 9 | 5 6 3 | 8 4 2 
------+-------+------
3 8 7 | 6 2 4 | 9 5 1 
6 9 1 | 3 5 8 | 7 2 4 
4 5 2 | 9 1 7 | 3 6 8 

Algorithm: BFS
Total time: 139 ms
Total expanded states: 20976
